Utility/Utility.Static/CMakeLists.txt

```diff
@@ -2,10 +2,9 @@ set(TARGET Utility.Static)
 set(OUTPUT_NAME Utility.Static)
 
 set(SOURCE_FILES
-    JsonSerializer.hpp
-    JsonSerializer.cpp)
+        empty.cpp
```

Comment:
You may avoid to creation empty. Since we do not use the static part, it is enough just to remove any mention and calls in CMake code. That lets us avoid a problem with .lib absence